src/interface/windows/mod.rs

```diff
@@ -51,7 +51,7 @@ impl Window {
     }
 
     pub fn has_transparency(&self, theme: &InterfaceTheme) -> bool {
-        self.get_background_color(theme).alpha != 255
+        self.get_background_color(theme).alpha_as_u8() != 255
```

Comment:
I think it would make sense here to compare against `1.0` so we don't bring floating point inaccuracy into the calculation

Reply:
Wouldn't it make more sense to compare using a small epsilon?
You are totally right that this isn't a good check, but i am not confident that comparing to 1.0 is a better approach.
I'll make a change and you just tell me if you are happy with it or not :)

Reply:
Very true!
I like the solution a lot :+1: